tests/InferTypesTest.elm

```diff
@@ -286,6 +302,23 @@ typeToString =
                 , "Maybe.Maybe a"
                 )
             ]
+        , describe "records"
+            [ runTest
+                ( "empty record"
+                , Record Dict.empty
+                , "{}"
+                )
+            , runTest
+                ( "one field record"
+                , Record <| Dict.fromList [ ( "a", Int ) ]
+                , "{ a : Int }"
+                )
+            , runTest
+                ( "two fields record"
+                , Record <| Dict.fromList [ ( "a", Int ), ( "b", String ) ]
+                , "{ a : Int, b : String }"
```

Comment:
Can we just add one small test here about parametric types, to be sure?
`Record <| Dict.fromList [("foo", Var 0)]` :arrow_right: `{ foo : a }` or something like that.

Reply:
I've checked the rest of the PR so after this addition we can merge :tada: :slightly_smiling_face: